docs/release/trg-1/trg-1-07.md

```diff
@@ -0,0 +1,26 @@
+---
+title: TRG 1.07 - End User Manual
+sidebar_position: 1
+---
+
+| Status | Created     | Post-History                                          |
+|--------|-------------|-------------------------------------------------------|
+| Active | 22-Apr-2024 |                                                       |
+| Draft  | 21-Mar-2024 | Transferred from Release Acceptance Criteria into TRG |
+
+## Why
+
+End user manual is essential for providing user assistance and ensuring a positive user experience. The user of the application is the recipient of this documentation. Typical contents are description of functionality, steps that guide through critical processes and screenshots.
```

Comment:
In the other TRGs, like TRG 1.06 (see #825), this section always indicates if the section (in this case an end user manual) **must** be provided or **shall** be provided. I would also vote for adding this information here.

Reply:
This is part of the ## Description. Your referenced TRG doesn't say it has to exist or not, it only says it shall help operators.

I'm fine with aligning it across TRGs, but for the sake of getting this merged before my capacity runs out: Pls approve and lets discuss alignment of it ater

Reply:
But what is the correct answer to my question: Does this section relate to a _must_ or to a _shall_?

Reply:
It is a must. It is a must if there is an enduser involved.

Reply:
Ok, then this should be adjusted in the text.

Reply:
Its updated. Pls check

Reply:
Great, thank you!